CONTRIBUTING.md

```diff
@@ -1,49 +1,151 @@
+```
+
+As you tackle new issues, you'll want to be sure that you always start by working on the most recent code. The second command is to set an upstream. By pulling from it, you can sync up your fork's `master` with a parent repository's `master`. For this to work, you should make sure you're always committing to a branch, not `master`.
+
+Each time you want to start a new page:
+```
+git checkout master
+git pull upstream master
+git checkout -b some-branch-name
+```
+
+### 3. Translate and lint the page
 
-- Clone the forked repository with `git clone <your fork's url>` and create a branch with `git checkout -b some-branch-name`.
+Each PR should have:
+- 1 commit
+- 2~3 files changed (`[some-page].md`, `.remarkignore`, optionally `.local.dic`)
 
-- Keep your commit stack clean with one commit, for instance: `git commit -m "feat: translate guides/release/index.md`
+While you are working, keep your commit stack clean with one commit, for instance: `git commit -m "feat: translate guides/release/index.md`.
 
-- Once you're at the point that you'd like feedback, submit a Pull Request (new Pull Request button). Choose `master` for the base and your branch name for `compare`, then submit it!
+Once you have translated the `.md` page of your choice, you need to run the linter to make sure there's no typo:
+- Add the path to your page in `.remarkignore` (respect the alphabetical order for readability)
+- Run `npm run lint`
 
-- Your PR will be reviewed by another contributor, and then either merged or have changes requested.
+If there are errors because "a word is written twice", but it's expected in French (Je vais vous "faire faire" une jolie traduction), make sure to fix all the other errors of the same paragraph before adding `<!-- spell ignore -->` above the paragraph.
 
-You are free to use the method you like to translate the file. Here are a few recommendations:
+If there are errors because you kept an English term commonly used in French, add this word the the `.local.dic` file (respect the alphabetical order for readability, lowercase words are below uppercase acronyms).
 
-- Don't translate variable names and don't modify code blocks. Most French-speaker developers have their code base in English, the purpose of this translation is to explain Ember concepts in a more accessible language.
+### 4. Label your PR
 
-- Keep the translation as close as possible to the original one and respect the original characters' case.
+Once you're at the point that you'd like feedback: 
+- Submit a Pull Request
+- Add the label "**Guides FR trad**" on it
 
-- [Google translate](https://translate.google.fr/?hl=fr&sl=en&tl=fr&op=translate) is an efficient translation tool that can set up an approximative translation for you, then you can fix the mistakes with your own words.
+## What syntax rules should I follow?
 
-## Keeping your fork updated
+### 1. Don't translate the code / translate the UI data
```

Comment:
```suggestion
### 1. Don't translate the code but translate the UI data
```

Reply:
I used slash as a visual opposition, I feel like using a full sentence with only words make both sides less visible

Reply:
In English, the word `but` is used to connect ideas that contrast, it expresses ideas in opposition.
The reader will grasp that opposition by reading "this way, BUT that way".

We can also simply be straightforward and just say

```
Don't translate the code, translate the UI data
```

Reply:
Besides, I forgot to add, but the very first time I read that title, I instinctively read it this way, hence my suggestions for all these titles:

```text
Don't translate the code / translate the UI data
      ˆˆˆˆ|ˆˆˆˆ            ˆˆˆˆ|ˆˆˆˆ
           ¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯

Don't translate the code
      translate the UI data
```

Some people unfamiliar with the process may read this as a "or".
Of course, immediately after I knew this meaning was wrong but because I am part of this repository.

To me, using a slash `/` does not ensure everyone will understand the same thing, the real meaning, words do.

---

Another nice and simple alternative:

```
Translate the UI data, not the code
.
Translate the UI data, but not the code
```

This better ensures readers will first see what they need to do.

```
Translate articles description, not their titles
.
Translate articles description, but not their titles
```